frontend/src/Store/Actions/discoverMovieActions.js

```diff
@@ -317,6 +317,18 @@ export const defaultState = {
         return tagList.sort(sortByName);
       }
     },
+    {
+      name: 'isAvailable',
+      label: translate('Is Available'),
```

Comment:
Translate keys wont have spaces in them

`src\NzbDrone.Core\Localization\Core\en.json` is where you would check for existing keys or add new ones as needed